ezsystems/ezplatform-ee-installer
# EZP-31478: Dropped LSE FullText index from clean EE data

| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-31478](https://jira.ez.no/browse/EZP-31478)
| **Requires**| ezsystems/ezpublish-kernel#2984
| **New feature**    | no
| **Target version** | `master` for eZ Platform EE `v3.0`
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

This PR drops LSE FullText index from clean EE data, see ezsystems/ezpublish-kernel#2984 for all the details

#### QA
Almost the same sanity checks as in ezsystems/ezpublish-kernel#2984, but please make sure to use `ezplatform-ee-clean` installer type for `ezplatform:install` command.
